core/services/workflows/monitoring.go

```diff
@@ -13,10 +13,18 @@ import (
 )
 
 var registerTriggerFailureCounter metric.Int64Counter
+var triggerWorkflowStarterErrorCounter metric.Int64Counter
 var workflowsRunningGauge metric.Int64Gauge
 var capabilityInvocationCounter metric.Int64Counter
+var capabilityFailureCounter metric.Int64Counter
+var workflowRegisteredCounter metric.Int64Counter
+var workflowUnregisteredCounter metric.Int64Counter
+var workflowExecutionFinishedCounter metric.Int64Counter
```

Comment:
We can observe this directly in the dashboard (will share link offline) by status already. What does execution finished vs WorkflowsRunningGauge tell us?